specification/mobilenetwork/resource-manager/Microsoft.MobileNetwork/preview/2022-01-01-preview/examples/AttachedDataNetworkCreate.json

```diff
@@ -0,0 +1,133 @@
+{
+  "parameters": {
+    "api-version": "2022-01-01-preview",
+    "subscriptionId": "subid",
+    "resourceGroupName": "rg1",
+    "packetCoreControlPlaneName": "TestPacketCoreCP",
+    "packetCoreDataPlaneName": "TestPacketCoreDP",
+    "attachedDataNetworkName": "TestAttachedDataNetwork",
+    "parameters": {
+      "location": "eastus",
+      "properties": {
+        "n6Interface": {
```

Comment:
@satravi I was expecting the N6 interface to be renamed as well, I think it has a different name in 4G

Reply:
@rainbowFi N6 is a common interface for both 4G and 5G - It is an interface between UPF and Data Network. So, we can keep the same existing parameter name.

Reply:
In the 4G standards, the interface between the LTE core and the public network is called the SGi interface. The N* names are all from the 5G standards.

Reply:
@rainbowFi Have updated N6 Interface attribute as dataNetworkInterface.